federation/manifests/federation-apiserver-deployment.yaml

```diff
@@ -42,9 +42,9 @@ spec:
           mountPath: /srv/kubernetes/
           readOnly: true
       - name: etcd
-        image: quay.io/coreos/etcd:v2.3.3
+        image: gcr.io/google_containers/etcd:3.0.14-alpha.1
         command:
-          - /etcd
+          - /usr/local/bin/etcd
```

Comment:
Fair enough, to get the tests passing. But I see that you made `etcd2` the default backend in `kubefed init`, but I don't see the corresponding change in this manifest. Our current test infra still uses this manifest. Shouldn't there be an equivalent change in this file?